app/src/main/java/com/eveningoutpost/dexdrip/Services/XDripViewer.java

```diff
@@ -25,7 +27,11 @@
 //need synchronization
 
 public class XDripViewer extends AsyncTaskBase {
-    
+
+    static Sensor sensor_exists = null; // Will hold reference to any existing sensor, to avoid looking in DB
+    static Boolean isNightScoutMode = null;
+    public final static String NIGHTSCOUT_SENSOR_UUID = "c5f1999c-4ec5-449e-adad-3980b172b921";

```

Comment:
What exactly is this String/ID? Is it always the same in certain cases? What cases are that?